agoradatatools/process.py

```diff
@@ -76,6 +81,15 @@ def process_dataset(dataset_obj: dict, syn=None):
 
 
 def create_data_manifest(parent=None, syn=None) -> DataFrame:
+    """Creates data manifest (dataframe) that has the IDs and version numbers of child synapse folders
+
+    Args:
+        parent (synapseclient.Folder, optional): _description_. Defaults to None.
```

Comment:
Nit: Missing description here.
Nit: The parent can also be a Synapse id which is a `str`

Reply:
added in latest commit